pkg/repo/mysql/services_integration_test.go

```diff
@@ -0,0 +1,213 @@
+package mysql
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/google/uuid"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+)
+
+var mysqlRepo *Repo
+
+func init() {
+	viper.SetEnvPrefix("translate_mysql")
+	viper.AutomaticEnv()
+
+	mysqlConf := Conf{
+		Addr:     viper.GetString("address"),
+		User:     viper.GetString("user"),
+		Database: viper.GetString("database"),
+	}
+
+	db, err := NewDB(context.Background(), &mysqlConf)
```

Comment:
why no context propagation?

Reply:
From where ? This is top level ctx

Reply:
might be worth to pass the context through the functions in this function scope, just looks :)

Reply:
Sure

Reply:
I suppose they should have tracing enabled